src/style.css

```diff
@@ -616,6 +673,106 @@ input:-moz-placeholder {
     font-size: 20px;
   }
 
+  #name {
+    font-size: 30px;
+    margin-top: 5px;
+  }
+
+  .popup-container {
+    background-color: white;
+  }
+
+  .close {
+    float: right;
+    margin-top: -25px;
+  }
+
+  .desk-button {
+    margin-top: -18px;
+  }
+
+  .Popup-image {
+    width: 90%;
+    margin-left: 5%;
+    margin-top: -25px;
+  }
+
+  .mob-pop {
+    /* display: flex; */
+    margin-right: 5px;
+    column-gap: 15%;
+    margin-left: 5%;
+  }
+
+  .skill {
+    display: flex;
+    margin: 4.5%;
+    padding-top: -6%;
+    margin-top: -7px;
+  }
+
+  .project-text {
+    margin-left: 5%;
+  }
+
+  .skill li {
+    margin-left: 8px;
+    font-family: "Inter", sans-serif;
+    padding: 10px;
+    font-size: 12px;
+    font-weight: 500;
+    font-style: normal;
+    color: black;
+    border: #000 solid;
+    border-width: 0.5px;
+    background-color: white;
+  }
+
+  #mobile {
+    background-color: white;
+    border: solid black;
+    border-width: 1px;
+    backdrop-filter: blur(2em);
+    overflow-y: scroll;
+  }
+
+  .mob-img {
+    display: flex;
+    flex-direction: column;
+    max-width: 100%;
+    height: auto;
+    row-gap: 10px;
+    margin: 5%;
+    margin-top: -10px;
+  }
+
+  .Pop-Img {
+    width: 100%;
+  }
+
+  .button-pop {
+    margin: 5%;
+    display: flex;
+    font-size: 50px;
+    padding-bottom: 30px;
+    column-gap: 4%;
+  }
```

Comment:
Great design team🤝

- [ ] As stated in the project requirements, we are supposed to stick to the Figma design provided so that our design can replicate it. Kindly implement all button interactions of `hover`, `enabled`, `disabled`, and `pressed`.
![buttons](https://github.com/Zel-hub7/Personal--Portfolio/assets/63917725/e0d6fe55-7d7f-4810-af86-02f15d5d9d76)